content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md

```diff
@@ -33,7 +33,7 @@ Try the steps below to [enable](/clusters/nested-clusters/cluster-quickstart#ena
 
   There are two ways to engage with a Nested Cluster on Palette.<p></p><br />
 
-   1. When you [create](/clusters/new-clusters) and [deploy](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster) a new Host Cluster, there is an option to **Enable Nested Clusters**. <p></p><br />Refer to the [Cluster Endpoint](/clusters/nested-clusters/cluster-quickstart#loadbalancer) step below for more information on how to configure this option. The same settings listed below are available, when you are enabling a Nested Cluster and when deploying a Host Cluster deployment wizard.
+   1. When you [create](/clusters/new-clusters) and [deploy](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster) a new Host Cluster, there is an option to **Enable Nested Clusters**. <p></p><br />Refer to the [Cluster Endpoint](/clusters/nested-clusters/cluster-quickstart#loadbalancer) step below for more information on how to configure this option. The same settings listed below are available, when you are enabling a Nested Cluster and when using the Host Cluster deployment wizard.
 
```

Comment:
Refer to the [Cluster Endpoint](https://docs.spectrocloud.com/clusters/nested-clusters/cluster-quickstart#loadbalancer) step below for more information on how to configure this option. The same settings listed below are available, when you are enabling a Nested Cluster and when deploying a Host Cluster deployment wizard.


here ' this option ' refers to?
the whole paragraph is slightly confusing. @Siboney1

Reply:
The "option" is mentioned in the previous sentence. the process is the same for both Nested and Host cluster deployment. Very clear.

Reply:
Tyler updated this section to clarify.